Python/instrumentation.c

```diff
@@ -458,6 +434,9 @@ dump_instrumentation_data(PyCodeObject *code, int star, FILE*out)
 static bool
 valid_opcode(int opcode)
 {
+    if (opcode == INSTRUMENTED_LINE) {
+        return true;
+    }
```

Comment:
It would be better to make the generator add INSTRUMENTED_LINE to _PyOpcode_opcode_metadata

Reply:
It would be, but this PR needs backporting to 3.12 and it's already quite large for a backport this late in the release cycle.